generator/feature_sorter.cpp

```diff
@@ -233,27 +242,61 @@ class FeaturesCollector2 : public FeaturesCollector
         }
       }
     }
+    }
+
+    // Override "alt_name" with synonym for Country or State for better search matching.
```

Comment:
Is it a hack?

Reply:
Actually, no. But, will add todo if we will decide to process OSM's short_name.